src/Vts.Test/MonteCarlo/DataStructuresValidation/TissueInputs/SingleInfiniteCylinderTissueInputValidationTests.cs

```diff
@@ -0,0 +1,205 @@
+using System.Collections.Generic;
+using NUnit.Framework;
+using Vts.Common;
+using Vts.MonteCarlo;
+using Vts.MonteCarlo.Detectors;
+using Vts.MonteCarlo.Sources;
+using Vts.MonteCarlo.Tissues;
+
+namespace Vts.Test.MonteCarlo.DataStructuresValidation.TissueInputs
+{
+    [TestFixture]
+    public class SingleInfiniteCylinderTissueInputValidationTests
+    {        
+        /// <summary>
+        /// Test to check that underlying MultiLayerTissue is good
+        /// </summary>
+        [Test]
+        public void validate_underlying_multilayer_tissue_definition()
+        {
+            var input = new SimulationInput(
+                10,
+                "",
+                new SimulationOptions(),
+                new DirectionalPointSourceInput(),
+                new SingleInfiniteCylinderTissueInput(
+                    new InfiniteCylinderTissueRegion(
+                        new Position(0, 0, 10),
+                        2.0,
+                        new OpticalProperties()),
+                    // define layer tissues that are incorrect
+                    new ITissueRegion[]
+                    {
+                        new LayerTissueRegion(
+                            new DoubleRange(double.NegativeInfinity, 0.0),
+                            new OpticalProperties(0.0, 1e-10, 1.0, 1.0)),
+                        new LayerTissueRegion(
+                            new DoubleRange(0.0, 20.0),
+                            new OpticalProperties(0.01, 1.0, 0.8, 1.4)),
+                        new LayerTissueRegion(
+                            new DoubleRange(100.0, double.PositiveInfinity),
+                            new OpticalProperties(0.0, 1e-10, 1.0, 1.0))
+                    }
+                ),
+                new List<IDetectorInput>() 
+                {
+                    new FluenceOfXAndYAndZDetectorInput()
+                }
+            );
+            var result = SimulationInputValidation.ValidateInput(input);
+            Assert.IsFalse(result.IsValid);
+        }
```

Comment:
This is nitpicky but I like to see a space between methods and therefore a line feed after the method and before the comments for the next method. I use space to help me find the individual methods.